python/rpdk/typescript/templates/package.json

```diff
@@ -8,7 +8,7 @@
         "dist"
     ],
     "scripts": {
-        "build": "npx tsc",
+        "build": "npx tsc --skipLibCheck",
         "prepack": "npm run build",
         "test": "echo \"Error: no test specified\" && exit 1"
     },
```

Comment:
lib_path is still set to 1.0.1 and needs to be updated to 1.0.6 in the codegen.py file